### PR TITLE
Mobile content revise

### PR DESCRIPTION
**Pull request by tonyphn**
Revise mobile content to be more inline with other discipline levels:
- Temporarily remove the mobile summary section until coming up with a better description.
- Simplify some points
- Add a few more common points to be inline with other disciplines.

### CODE REVIEW

File: frameworks/engineering/mobile.md [hunk 1]
```diff
@@ -7,213 +7,203 @@ yaml: true
...
+          - "Write code according to our style guides and architecture"
           - "Proactive in asking questions, explains what they have tried so far and why that hasn’t worked"
-          - "Updates layout to existing screens"
-          - "Builds simple screens or flows"
-          - "Asks questions and actions feedback from PRs and other engineers"
-          - "Fixes simple bugs or issues"
+          - "Update layout to existing screens"
+          - "Build simple screens or flows"
+          - "Ask questions and actions feedback from PRs and other engineers"
+          - "Fixe simple bugs or issues"
```
Comment (Prav-Patel) [inline frameworks/engineering/mobile.md:145]: Fix

File: frameworks/engineering/mobile.md [hunk 2]
```diff
@@ -7,213 +7,203 @@ yaml: true
...
         exampleCriteria:
-          - criteria: "Integrates with new APIs"
+          - criteria: "Integrate with new APIs"
             examples:
               - "Can add an integration with a new backend endpoint and handle return data"
               - "Can add an integration with a documented third party API"
           - criteria: "Uses concurrency appropriately"
             examples:
               - "Can reason whether to use asynchronous or synchronous operations"
-              - "Understands impact on app performance to use either"
```
Comment (Prav-Patel) [inline frameworks/engineering/mobile.md:176]: This was right `Understands `

File: frameworks/engineering/mobile.md [hunk 3]
```diff
@@ -238,7 +228,7 @@ The framework covers all the things we’re looking for from engineers at Bright
 - Comms & Feedback - How you interact with others
 - Leadership - How people around you become better and more impactful
 
-We sort them into four levels, and we try to give specific examples of behaviours we expect for each. Each of those levels has a fairly wide salary range associated with it, and within each level you can progress in sub-levels, labelled A–E. So even if you’re at level 3 for a couple of years, you’ll still be able to see that you’re moving forward. Basically, the more behaviours you show from your level, the more you’ll progress.
+We sort them into six levels, and we try to give specific examples of behaviours we expect for each. Each of those levels has a fairly wide salary range associated with it, and within each level you can progress in sub-levels, labelled A–E. So even if you’re at level 3 for a couple of years, you’ll still be able to see that you’re moving forward. Basically, the more behaviours you show from your level, the more you’ll progress.
```
Comment (Prav-Patel) [inline frameworks/engineering/mobile.md]: There is only 4 levels. 1-4

File: frameworks/engineering/mobile.md [hunk 4]
```diff
@@ -7,213 +7,203 @@ yaml: true
 levels: 4
 homepage: true
 topics:
-  - name: "summary"
```
Comment (Prav-Patel) [inline frameworks/engineering/mobile.md:10]: This was a good idea, why did you remove?

**Review by Prav-Patel — COMMENTED** [review]
I've made some comments, what made you make every point singular?

File: frameworks/engineering/mobile.md (hunk 4 above)
Comment (tonyphn) [reply frameworks/engineering/mobile.md:10]: I like it too just haven't found a good summary yet

File: frameworks/engineering/mobile.md (hunk 2 above)
Comment (tonyphn) [reply frameworks/engineering/mobile.md:176]: was meant to refer to you/they here

File: frameworks/engineering/mobile.md (hunk 3 above)
Comment (tonyphn) [reply frameworks/engineering/mobile.md]: I think you commented on a outdated change. It is four atm.

File: frameworks/engineering/mobile.md (hunk 1 above)
Comment (tonyphn) [reply frameworks/engineering/mobile.md]: Good spot, updated thanks.

**Comment by tonyphn** [comment]
> I've made some comments, what made you make every point singular?

There are still some points with examples. I only made some simple/obvious points singular.

**Review by LazyDave76 — CHANGES_REQUESTED** [review]
some spelling errors have crept in.  you seem to dislike the letter s :)

**Review by Prav-Patel — APPROVED** [review]

File: frameworks/engineering/mobile.md [hunk 5]
```diff
@@ -7,213 +7,203 @@ yaml: true
...
-          - "Anticipates large or significant platform changes, work with team to define possible solutions. Starts or finishes these projects."
-          - "Takes high level requirements and turns them into discrete technical concerns"
+          - "Anticipate large or significant platform changes, work with team to define possible solutions. Starts or finishes these projects."
+          - "Take high level requirements and turns them into discrete technical concerns"
         exampleCriteria:
-          - criteria: "Owns and coordinates large scale architectural changes to the entire codebase"
+          - criteria: "Own and coordinate large scale architectural changes to the entire codebase"
             examples:
-              - "Researches potential solutions and evaluates how they solve the problem."
+              - "Researche potential solutions and evaluates how they solve the problem."
```
[the remark this answers is not in the document]
Comment (tonyphn) [reply frameworks/engineering/mobile.md]: Nice spots updated thanks @LazyDave76

**Review by LazyDave76 — APPROVED** [review]

**Pull request merged**